loopdb/meta.go

```diff
@@ -31,11 +31,20 @@ var (
 	// of database don't match with latest version this list will be used
 	// for retrieving all migration function that are need to apply to the
 	// current db.
-	migrations = []migration{}
+	migrations = []migration{noMigrationAvailable}
 
 	latestDBVersion = uint32(len(migrations))
 )
 
+// noMigrationAvailable is the fall back migration in case there is no migration
+// implemented.
+func noMigrationAvailable(tx *bbolt.Tx) error {
+	return errors.New(
+		"migration not implemented, consider removing swap db after " +
```

Comment:
Why not just do a migration filling in zeros for the prior costs? This way the users don't need to wipe their exiting state which they may want to retain for accounting reasons.

Reply:
My thought was it isn't worth the effort given we are still in alpha. But added.